decidim-comments/app/models/decidim/comments/comment.rb

```diff
@@ -92,7 +92,10 @@ def reported_content_url
 
       # Public: Returns the comment message ready to display (it is expected to include HTML)
       def formatted_body
-        @formatted_body ||= Decidim::ContentProcessor.render(sanitized_body)
+        @formatted_body ||= begin
+          text = Decidim::ContentProcessor.render(sanitized_body)
+          Anchored::Linker.auto_link(text, target: "_blank", rel: "noopener")
```

Comment:
@aitorlb It would be great to extract this feature into a ContentProcessor so that it can be reused by proposals, and make its users abstracted from Anchored::Linker, as we said.